ui/public/i18n/en/translation.json

```
@@ -56,6 +56,24 @@
     "smarthost_is_disabled": "Email notifications are disabled",
     "smarthosts_is_needed_to_send_notifications": "Enable Smarthost to receive CrowdSec email notifications"
   },
+  "unban":{
+    "title": "Banned IP addresses",
+    "unban_tooltip": "IP addresses that failed multiple login attempts are listed here. You can unban them if needed",
+    "no_bans": "No active bans",
+    "reload_bans": "Refresh",
+    "delete_all": "Unban all",
+    "delete": "Unban",
+    "search_bans": "Search",
+    "col_created_at": "Banned",
+    "col_value": "IP address",
+    "col_duration": "Ban time",
+    "col_scenario": "Reason",
+    "unban_ip": "Unban IP",
+    "confirm_unban_ip_message": "The IP will be released are you sure ?",
```

Let's always use the same glossary

```suggestion
    "confirm_unban_ip_message": "Unban IP address?",
```